static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -1,42 +1,91 @@
 import React from 'react';
+import styled from '@emotion/styled';
 
-import EventEntry from 'sentry/components/events/eventEntry';
-import {Entry, Event} from 'sentry/types/event';
-import useOrganization from 'sentry/utils/useOrganization';
-import {useRouteContext} from 'sentry/utils/useRouteContext';
+import Category from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/category';
+import Time from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/time';
+import Type from 'sentry/components/events/interfaces/breadcrumbs/breadcrumb/type';
+import {transformCrumbs} from 'sentry/components/events/interfaces/breadcrumbs/utils';
+import {
+  Panel as BasePanel,
+  PanelBody as BasePanelBody,
+  PanelHeader as BasePanelHeader,
+  PanelItem,
+} from 'sentry/components/panels';
+import space from 'sentry/styles/space';
+import {RawCrumb} from 'sentry/types/breadcrumbs';
+import {Event} from 'sentry/types/event';
 
 type Props = {
-  entry: Entry | undefined;
+  crumbs: RawCrumb[];
   event: Event | undefined;
 };
 
-function UserActionsNavigator({event, entry}: Props) {
-  // TODO(replay): New User Actions widget replaces this breadcrumb view
-
-  const {routes, router} = useRouteContext();
-  const organization = useOrganization();
-
-  if (!entry || !event) {
+function UserActionsNavigator({event, crumbs}: Props) {
+  if (!event) {
     return null;
   }
 
-  const projectSlug = getProjectSlug(event);
+  const transformedCrumbs = transformCrumbs(crumbs);
 
   return (
-    <EventEntry
-      projectSlug={projectSlug}
-      // group={group}
-      organization={organization}
-      event={event}
-      entry={entry}
-      route={routes[routes.length - 1]}
-      router={router}
-    />
+    <Panel>
+      <PanelHeader>Event Chapters</PanelHeader>
+
+      <PanelBody>
+        {transformedCrumbs.map(item => (
+          <PanelItemCenter key={item.id}>
+            <Wrapper>
+              <Type type={item.type} color={item.color} description={item.description} />
+              <Category category={item.category} searchTerm="" />
+            </Wrapper>
+            {/* Probably we need to use or create another component to show the timestamp relative */}
+            <Time
+              timestamp={event.dateReceived}
```

Comment:
oops, i didn't use the correct example before. but instead of `dateReceived` what about `startTimestamp`?